scripts/question_answering/question_id_mapper.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Question id mapper to and from int"""
+
+
+class QuestionIdMapper:
+    """Stores mapping between question id and context of SQuAD dataset
+
+    """
+    def __init__(self, dataset):
+        self._question_id_to_context = {item[1]: item[3] for item in dataset}
+        self._question_id_to_idx = {item[1]: item[0] for item in dataset}
```

Comment:
This assumes a schema that wasn't documented.

Reply:
Well, it says above, that it uses SQuAD dataset as an argument. The schema of SQuAD dataset is described in `gluonnlp/data/question_answering.py`. 

We certainly can improve it by providing more comments, if needed.

Reply:
Added pydoc to the __init__ method that dataset is a SQuAD dataset